FightPandemics/Features/Post/CreatePost/PostCreationSuccessAlert.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  PostCreationSuccessAlert.swift
+//  FightPandemics
+//
+//  Created by Onyekachi Ezeoke on 29/05/2020.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class PostCreationSuccessAlert: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+
+    private struct Dimension {
+        static let viewPostButtonHeight: CGFloat = 48
+    }
+
+    static let shared = PostCreationSuccessAlert()
+
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private let containerView = UIView()
+    private let imageView = UIImageView(image: #imageLiteral(resourceName: "connectivity"))
+    private let titleLabel = UILabel()
+    private let messageLabel = UILabel()
+    private let viewPostButton = UIButton()
+
+    private func setupViews() {
+        backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 0.2727686216)
+        containerView.backgroundColor = .white
+        containerView.layer.cornerRadius = 10
+        containerView.layer.masksToBounds = true
+
+        imageView.contentMode = .scaleAspectFill
+
+        titleLabel.textColor = #colorLiteral(red: 0.1568627451, green: 0.1568627451, blue: 0.1568627451, alpha: 1)
```

Comment:
All colors are available in the palette, are they different or you just wanted to use these colors? (they seems to be more precise though, I'm just asking)

Reply:
Thanks Ethan, I will check that out and update accordingly.

Reply:
I will be maintaining this for now, just like you said, this one is more precise and accurate, the quality is lost when I tried using the color functions we have already.